addons/SpellCheck/SpellCheck.lua

```diff
@@ -67,33 +69,33 @@ end
 
 --Get spells
 function display_spell_count(command)
+
+    missing_spells_len = 0
     missing_spell_names = {}
     
     --Get all, current and missing spells 
     all_spells = res.spells:type(spell_type[command]):keyset()
-    current_spells = T(windower.ffxi.get_spells()):filter(boolean._true):keyset()
+    current_spells = T(windower.ffxi.get_spells()):filter(boolean._true):keyset()   
+    
     missing_spells = all_spells - current_spells
     current_spells = all_spells * current_spells
-    
-    --Get count of all, current and missing spells 
-    all_spells_len = all_spells:length()
-    current_spells_len = current_spells:length()
-    missing_spells_len = 0
-    uc_trust_spells_len = 0
-    
+        
     --Add missing spells to table for sorting
     for spell in missing_spells:it() do
-        --if trusts are being searched, exclude Unity trusts
-        if command == "tru" then
-            if not string.match(res.spells[spell].name, "(UC)") then
+        --Trust and spells must be processed separately
+        if command == 'tru' then
+            --Only include non Unity trusts
+            if not string.match(res.spells[spell].name, '(UC)') then
```

Comment:
This works just fine, but personally I prefer being more specific when possible. As such I'd do:
```lua
            if not res.spells[spell].name:endswith('(UC)') then
                -- Stuff
            end
```